tutorials/dataframe/df017_vecOpsHEP.py

```diff
@@ -11,13 +11,12 @@
 ## \author Danilo Piparo, Andre Vieira Silva
 
 import ROOT
-from math import sqrt
 
 filename = ROOT.gROOT.GetTutorialDir().Data() + "/dataframe/df017_vecOpsHEP.root"
 treename = "myDataset"
-RDF = ROOT.ROOT.RDataFrame
 
-def WithPyROOT():
+def WithPyROOT(filename):
+    from math import sqrt
```

Comment:
Why moving the import inside the function?

Reply:
Same here. The import if done at the file scope is not visible in the local scope of WithPyROOT() when the tutorial is run inside the makeimage wrapper.